.env

```diff
@@ -10,12 +10,26 @@ API_PROVIDER_URL_1=
 API_PROVIDER_URL_4=
 API_PROVIDER_URL_5=
 
+# Subgraph endpoint
+SUBGRAPH_MAINNET=https://api.thegraph.com/subgraphs/name/lidofinance/lido
+SUBGRAPH_ROPSTEN=
```

Comment:
I think we need to shorten the list of supported chains. Not all of them are already working.

Reply:
More context pls

Reply:
only mainnet and goerli